huaweicloud/cpi.yml

```diff
@@ -0,0 +1,94 @@
+---
+- type: replace
+  path: /releases/-
+  value:
+    name: bosh-huaweicloud-cpi
+    version: "2"
+    url: https://obs-bosh.obs.otc.t-systems.com/bosh-huaweicloud-cpi-release-r2.tgz
+    sha1: 179e71e907a4ac7a5b04f13ae5c9c6fdd6fbad24
+
+- type: replace
+  path: /resource_pools/name=vms/stemcell?
+  value:
+    url: https://obs-bosh.obs.otc.t-systems.com/bosh-stemcell-2.0-huaweicloud-xen-ubuntu-trusty-go_agent.tgz
+    sha1: 0584dd353988593304a6b81f54380578d3341337
+
+# Configure sizes
+- type: replace
+  path: /resource_pools/name=vms/cloud_properties?
+  value:
+    instance_type: s2.large.2
+    availability_zone: ((az))
+
+- type: replace
+  path: /networks/name=default/subnets/0/cloud_properties?
+  value: {subnet_id: ((subnet_id))}
+
+# Enable registry job
```

Comment:
@zhongjun2, we noticed that you're enabling the registry job in your CPI. Currently, the BOSH team is working on deprecating the registry and we wanted to make sure that you're aware of this change. If the Huawei Cloud already supports registry-less operation, it would be ideal to remove these blocks.